src/redux/store.js

```diff
@@ -0,0 +1,13 @@
+import { configureState } from '@reduxjs/toolkit';
```

Comment:
- [ ] The correct module to create a reducer should be redux store should be `configureStore` instead of `configureState`. Let's ensure to improve upon this aspect.